101-confidential-ledger-create/azuredeploy.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "ledgerName": {
+            "type": "string",
+            "metadata": {
+                "description": "Ledger Name"
+            },
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "principalId": {
+            "type": "string",
+            "metadata": {
+                "description": "Oid of the user"
+            },
+            "minLength": 18,
+            "maxLength": 48
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "functions": [],
+    "variables": {},
+    "resources": [{
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS code)

Reply:
JSOn has been fomatted